consensus/polybft/event_tracker_test.go

```diff
@@ -74,7 +76,9 @@ func TestEventTracker_TrackSyncEvents(t *testing.T) {
 
 	// send 10 more events
 	for i := 0; i < 10; i++ {
-		server.TxnTo(addr, "emitEvent")
+		receipt, err := server.TxnTo(addr, "emitEvent")
+		require.NoError(t, err)
+		require.True(t, receipt.Status == uint64(types.ReceiptSuccess))
```

Comment:
Does it worth it to use require.Equal ?

Reply:
https://github.com/0xPolygon/polygon-edge/pull/935/commits/26f468f78cf2a7ffe5d13931ddccbb2f0b8048f8